model/Human-GEM.yml

```diff
@@ -302273,7 +302273,7 @@
           - MAM02630m: -1
           - MAM02631m: 0.02
       - lower_bound: 0
-      - upper_bound: 1000
```

Comment:
Is it alright to leave the bound as it is before? I'm not certain which reaction will carry the flux, whether this ROS one or the regular one. I personally vote for keeping them both open. In the step of specific model generation, the user can remove either one of these options. Blocking them in the generic model could lead to unnecessary issues in the specific model simulation, making it difficult to identify.

Reply:
In my opinion, only one version should be left unblocked by default. Until this thread, it didn't even occur to me that Complex IV effectively has twice as much flux capacity as the rest of the ETC reactions by default, since the other ETC reactions don't (as far as I know) have comparable ROS-producing duplicate versions. Nothing about MAR06914 suggests that it has an ROS-producing duplicate in MAR13081, and I think it'd it'd be much harder to figure out why it seems like there's more flux than you bargained for going through Complex IV (e.g. you only changed the bound on MAR06914 and did not know MAR13081 existed) than to figure out why Complex IV isn't producing ROS (which you can figure out easily by just looking at the reaction string for MAR06914 and seeing no ROS production)

Reply:
I agree with @Devlin-Moyer, this reaction can easily be missed, and may cause confusion if one is not aware of the two pathways. I can't think of a scenario where deactivating it would cause any issues, but I can think of a few annoyances that could be caused by keeping both reactions open. So I would lean toward deactivating it.

Reply:
Also, at the moment, nobody seems to know where the stoichiometric coefficients in MAR13081 came from, but the stoichiometric coefficients in MAR06914 clearly come from [the associated reference](https://www.sciencedirect.com/science/article/pii/0304417386900066?via%3Dihub)

Reply:
Ok, in that case, could we change the name or the subsystem to indicate the reason why we block this at least point to ROS or similar commet. Otherwise, in one or two years, we forget why we block this rxn.

Reply:
@feiranl the reason the model is maintained on GitHub is precisely so that in 2 years (or more) we can search this repository to find out why changes were made. I would not recommend changing the subsystem just for tracking reasons, but if there is another subsystem which is actually deemed more appropriate, then of course that change can be considered. It's also fine to update the name of the reaction to be more descriptive, if necessary.